vocabularies/earth-science-data-category.ttl

```diff
@@ -98,6 +105,67 @@ esdc:geology a skos:Concept ;
     skos:notation "0403" ;
     skos:prefLabel "Geology"@en .
 
+esdc:regional-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The geological study of large-scale regions, providing a link between global and local geology, and a framework for site-specific studies. (Bally & Roberts, 2012)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Regional Geology"@en .
+
+esdc:general-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The study of many aspects of Earth Sciences, including rocks and minerals, geologic processes and hazards, and global-scale geologic systems. (Modified from: https://serc.carleton.edu/NAGTWorkshops/intro/courses/23694.html; 17.04.20020"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "General Geology"@en .
+
+esdc:history-of-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The historical development of geologic knowledge, including the history of observations of geologic features, geological exploration, drilling, mapping, etc. (Modified from: Neuendorf et al., 2011)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "History of Geology"@en .
+    skos:scopeNote "Also includes: The history of theories to explain the origin of geologic features, the history of the organisation and development of geologic institutions and societies, and the biographical study of geologists. Not to be confused with historical geology."@en ;
+
+esdc:historical-geology a skos:Concept ; 
+    skos:altLabel "paleogeology"@en , 
+        "palaeogeology"@en ;
+    skos:broader esdc:geology ; 
+    skos:definition "The branch of geology concerned with the reconstruction and comprehension of the geological history of the Earth. (Neuendorf et al., 2011)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Historical Geology"@en .
+    skos:scopeNote "Includes geologic processes on the surface of, and within, the Earth, involving investigations into stratigraphy, paleontology and the associated evolution of life forms throughout the interval of the geologic time scale, geochronology, plate tectonics, structural geology, paleoenvironments and changing paleoclimates. It is not to be confused with history of geology."@en ;
+
+esdc:areal-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The comprehensive study of an area, including geological and geographical features. (The Free Dictionary by Farlex, 06.05.2020)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Areal Geology"@en .
```

Comment:
Should this be Local Geology as referenced in the Regional Geology concept definition?
Or at least added as an altLabel

Reply:
@johnmckellar 
Can you please comment? I _think_ the intention here is to describe local geology.
So regional geology = "The geological study of large-scale regions, providing a link between global and local geology, and a framework for site-specific studies. (Bally & Roberts, 2012)."

then at the next level down

areal/local geology = "The comprehensive study of an area, including geological and geographical features. (The Free Dictionary by Farlex, 06.05.2020)."

If no can you please clarify.
If yes can you please review the following suggestion

```suggestion
esdc:areal-geology a skos:Concept ; 
    skos:broader esdc:geology ; 
    skos:definition "The comprehensive study of an area, including geological and geographical features. (The Free Dictionary by Farlex, 06.05.2020)."@en ;
    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
    skos:altLabel "Areal Geology"@en ;
    skos:prefLabel "Local Geology"@en  .
```

Reply:
Hi Vance,
Considering Dave and Bob's feedback, why don't we just make this local geology in the first place, and even forget about areal geology as an altLabel?

Reply:
Hi John. I've updated my suggestion, based on your suggestion.
Have kept areal geology as a hidden label... just out of paranoia for the moment.

Reply:
"Involves a comprehensive study of the geological and geographical features of an area."@en ;

There is no real definition out there, so I have made one up (based on the Free Dictionary, for areal geology). Local geology is not defined therein. So, if you want to modify the code for this, I'll run with it. Cheers.